subsys/mgmt/mcumgr/lib/cmd/fs_mgmt/Kconfig

```diff
@@ -125,8 +125,9 @@ config FS_MGMT_PATH_SIZE
 	  of this size gets allocated on the stack during handling of file upload
 	  and download commands.
 
-config FS_MGMT_FILE_ACCESS_HOOK
+config MGMT_MCUMGR_GRP_FS_FILE_ACCESS_HOOK
```

Comment:
Sorry for asking to change this again, but I have agreed that the prefix `MGMT_` is too much, and `MCUMGR_` is distinctive enough.